annotation-processing/src/main/java/com/exactpro/epfast/annotation/processing/FastFieldResolver.java

```diff
@@ -27,14 +27,12 @@ private boolean isOverriddenIn(FastTypeElement fastType, FastFieldElement fastFi
     }
 
     private List<FastFieldElement> getFastFieldsFor(Element aClass) {
-        if (cache.containsKey(aClass)) {
-            return cache.get(aClass);
-        }
-        List<FastFieldElement> fastFields = aClass.getEnclosedElements().stream()
+        List<FastFieldElement> fastFields = cache.computeIfAbsent(aClass,
+            key -> key.getEnclosedElements().stream()
             .filter(element -> element.getAnnotation(FastField.class) != null)
             .map(element -> new FastFieldElement((ExecutableElement) element))
-            .collect(Collectors.toList());
-        cache.put(aClass, fastFields);
+            .collect(Collectors.toList()));
+        cache.putIfAbsent(aClass, fastFields);
```

Comment:
Lasha, this is redundant. If you see something new to you (or if you doubt), don't be lazy and check the documentation. https://docs.oracle.com/javase/8/docs/api/java/util/Map.html#computeIfAbsent-K-java.util.function.Function-